ivy/stateful/module.py

```diff
@@ -832,6 +833,21 @@ def train(self, mode: bool = True):
             if isinstance(module, ivy.Module):
                 module.train(mode=mode)
 
+    def to_device(self, device, out=None):
```

Comment:
let's remove the `out` argument, ig most people would assume `model.to_device` to be inplace by default